client/dashboard/customizable.js

```diff
@@ -344,6 +365,32 @@ export default compose(
 					[ 'woocommerce_task_list_hidden' ],
 					'no'
 				) === 'yes';
+
+			const options = getOptions( [
+				'woocommerce_task_list_skip_store_setup',
+				'woocommerce_task_list_do_this_later',
+			] );
+
+			withSelectData.skipStoreSetup = get(
+				options,
+				[ 'woocommerce_task_list_skip_store_setup' ],
```

Comment:
Do you think we could just use `woocommerce_task_list_hidden` to avoid introducing a new option and new logic to other areas of onboarding?

Reply:
Nice, done